modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -75,19 +117,26 @@ export default function AuditBarChart({bars, data, rtl}) {
 						tick={<CustomYAxisTick rtl={rtl} />}
 						tickLine={false}
 					/>
-
-					{bars.map((bar, index) => {
-						return (
-							<Bar
-								barSize={BAR_CHART.barHeight}
-								dataKey={bar.dataKey}
-								fill={bar.fill}
-								key={index}
-								legendType="square"
-								name={bar.name}
-							/>
-						);
-					})}
+					{bars &&
+						bars.map((bar, index) => {
+							return (
+								<Bar
+									barSize={BAR_CHART.barHeight}
+									dataKey={bar.dataKey}
+									fill={COLORS[index % COLORS.length]}
+									key={index}
+									legendType="square"
+									name={bar.name}
+								/>
+							);
+						})}
+					{bars === undefined && (
```

Comment:
Same for here, might be better to check for falsy rather than a strict undefined. `!bars &&`